packages/react-query/src/hooks/tokens/useTokenSecurity.ts

```diff
@@ -1,73 +1,197 @@
-import { useQuery } from '@tanstack/react-query'
+import { type UseQueryOptions, useQuery } from '@tanstack/react-query'
 import { ChainId } from 'sushi/chain'
 import { Token } from 'sushi/currency'
+import { z } from 'zod'
 
-const SUPPORTED_CHAINS: ChainId[] = [
+const SUPPORTED_CHAIN_IDS = [
```

Comment:
This should probably be somewhere in the `sushi` pkg

Reply:
moved to `sushi/config`